packages/devtools_app/lib/src/inspector/flutter/inspector_screen.dart

```diff
@@ -87,6 +88,15 @@ class _InspectorScreenBodyState extends State<InspectorScreenBody>
 
   @override
   Widget build(BuildContext context) {
+    final summaryTree = Expanded(
```

Comment:
I would keep the summaryTree as
InspectorTree(controller: summaryTreeController);
and wrap it with an Expanded in the code building up the widget tree.

Reply:
now that you mention it, it seems more intuitive and elegant that way, will change!